deps/cares/src/ares_mkquery.c

```diff
@@ -191,5 +168,22 @@ int ares_mkquery(const char *name, int dnsclass, int type, unsigned short id,
   DNS_QUESTION_SET_TYPE(q, type);
   DNS_QUESTION_SET_CLASS(q, dnsclass);
 
+  q += QFIXEDSZ;
```

Comment:
In the latest version, this line was previously inside a `if (max_udp_size) {}` block but was lifted out for the patch. In this backport it's standing alone and I'm not super-confident that it belongs here.